### PR TITLE
ref(example): update loadCollection initialize action name

### LINKED ISSUE #2334: Fix loadCollection action in example app to fit good actions hygiene
## Context
We are discussing implementing Good Actions Hygiene from @MikeRyanDev in our project and found one place in example app that seems does not fit this pattern.

Here is the place: https://github.com/ngrx/platform/blob/master/projects/example-app/src/app/books/containers/collection-page.component.ts#L43

It's a common practice to dispatch some action in component's onInit. So, from actions-like-events standpoint type should be something like "[Collection Page] Initialization" (event from component, no idea how store will handle this event), but in example it sounds more like a command: "[Collection Page] Load Collection" (command to store what to do).

## Proposal
So, proposal is to fix loadCollection action to be collectionPageInitialization (or Init ?) with type "[Collection Page] Initialization".

Is that a right naming in such cases? 

Would be happy to add pr.

## If accepted, I would be willing to submit a PR for this feature

[X] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

### PR DESCRIPTION
**Pull request by jordanpowell88**
Closes #2334

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #2334

## What is the new behavior?

Renames action to improve action hygiene.

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
Go ahead and rename the action creator also to `enter`

File: projects/example-app/src/app/books/actions/collection-page.actions.ts [hunk 1]
```diff
@@ -3,4 +3,4 @@ import { createAction } from '@ngrx/store';
 /**
  * Load Collection Action
  */
-export const loadCollection = createAction('[Collection Page] Load Collection');
+export const loadCollectionEnter = createAction('[Collection Page] Enter');
```
Comment (brandonroberts) [inline projects/example-app/src/app/books/actions/collection-page.actions.ts]:
```suggestion
export const enter = createAction('[Collection Page] Enter');
```

File: projects/example-app/src/app/books/containers/collection-page.component.spec.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ describe('Collection Page', () => {
   });
 
   it('should dispatch a collection.Load on init', () => {
-    const action = CollectionPageActions.loadCollection();
+    const action = CollectionPageActions.loadCollectionEnter();
```
Comment (timdeschryver) [inline projects/example-app/src/app/books/containers/collection-page.component.spec.ts]:
```suggestion
    const action = CollectionPageActions.enter();
```

File: projects/example-app/src/app/books/containers/collection-page.component.ts [hunk 3]
```diff
@@ -40,6 +40,6 @@ export class CollectionPageComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.store.dispatch(CollectionPageActions.loadCollection());
+    this.store.dispatch(CollectionPageActions.loadCollectionEnter());
```
Comment (timdeschryver) [inline projects/example-app/src/app/books/containers/collection-page.component.ts]:
```suggestion
    this.store.dispatch(CollectionPageActions.enter());
```

File: projects/example-app/src/app/books/effects/collection.effects.spec.ts [hunk 4]
```diff
@@ -62,7 +62,7 @@ describe('CollectionEffects', () => {
   });
   describe('loadCollection$', () => {
     it('should return a collection.LoadSuccess, with the books, on success', () => {
-      const action = CollectionPageActions.loadCollection();
+      const action = CollectionPageActions.loadCollectionEnter();
```
Comment (timdeschryver) [inline projects/example-app/src/app/books/effects/collection.effects.spec.ts]:
```suggestion
      const action = CollectionPageActions.enter();
```

File: projects/example-app/src/app/books/effects/collection.effects.spec.ts [hunk 5]
```diff
@@ -76,7 +76,7 @@ describe('CollectionEffects', () => {
     });
 
     it('should return a collection.LoadFail, if the query throws', () => {
-      const action = CollectionPageActions.loadCollection();
+      const action = CollectionPageActions.loadCollectionEnter();
```
Comment (timdeschryver) [inline projects/example-app/src/app/books/effects/collection.effects.spec.ts]:
```suggestion
      const action = CollectionPageActions.enter();
```

File: projects/example-app/src/app/books/effects/collection.effects.ts [hunk 6]
```diff
@@ -31,7 +31,7 @@ export class CollectionEffects {
 
   loadCollection$ = createEffect(() =>
     this.actions$.pipe(
-      ofType(CollectionPageActions.loadCollection),
+      ofType(CollectionPageActions.loadCollectionEnter),
```
Comment (timdeschryver) [inline projects/example-app/src/app/books/effects/collection.effects.ts]:
```suggestion
      ofType(CollectionPageActions.enter),
```

File: projects/example-app/src/app/books/reducers/collection.reducer.ts [hunk 7]
```diff
@@ -21,7 +21,7 @@ const initialState: State = {
 
 export const reducer = createReducer(
   initialState,
-  on(CollectionPageActions.loadCollection, state => ({
+  on(CollectionPageActions.loadCollectionEnter, state => ({
```
Comment (timdeschryver) [inline projects/example-app/src/app/books/reducers/collection.reducer.ts]:
```suggestion
  on(CollectionPageActions.enter, state => ({
```

**Pull request merged**